package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.10-prerelease.1",
```

Comment:
don't you mean 1.7.11?

Reply:
(though we should have already been on 11-prerelease technically)

Reply:
Ah yup, I had to temporarily change the version to `1.7.10` to get `make release-manual` to work.